test/axios.test.js

```diff
@@ -80,15 +58,117 @@ describe('axios module', () => {
   })
 
   test('ssr no brotli', async () => {
-    const makeReq = login =>
-      axios
-        .get(url('/ssr' + (login ? '?login' : '')))
-        .then(r => r.data)
-        .then(h => /encoding-\$(.*)\$/.exec(h))
-        .then(m => (m && m[1] ? m[1] : null))
+    const makeReq = login => axios
+      .get(url('/ssr' + (login ? '?login' : '')))
+      .then(r => r.data)
+      .then(h => /encoding-\$(.*)\$/.exec(h))
+      .then(m => (m && m[1] ? m[1] : null))
 
     const result = await makeReq()
 
     expect(result).toBe('gzip, deflate')
   })
+}
+
+describe('module', () => {
+  beforeAll(async () => {
+    nuxt = new Nuxt(config)
+
+    // Spy addTemplate
+    addTemplate = nuxt.moduleContainer.addTemplate = jest.fn(
+      nuxt.moduleContainer.addTemplate
+    )
+
+    await new Builder(nuxt).build()
```

Comment:
This `build()` call is costly for each test and unnecessary for just coverage. I suggest using `generateRouteAndFiles`. Have to look into the nuxt code to point to an example.

Reply:
@pi0 I think it got better and faster